applications/mne_x/plugins/rthpi/rthpi.cpp

```diff
@@ -254,29 +254,24 @@ void RtHpi::run()
     coil.pos = Eigen::MatrixXd::Zero(numCoils,3);
     coil.mom = Eigen::MatrixXd::Zero(numCoils,3);
 
-    int numCh = m_pFiffInfo->nchan;
     //
     // Read Fiff Info
     //
     while(!m_pFiffInfo) msleep(10);// Wait for fiff Info
+    int numCh = m_pFiffInfo->nchan;
 
     m_bProcessData = true;
 
     QVector<int> refind(0); // Reference Indices
 
     for(int j=0;j<numCh;j++) {
-        if (m_pFiffInfo->ch_names[j].indexOf("TRG_013") >= 0) {
-           refind.append(j);
-        }
-        if (m_pFiffInfo->ch_names[j].indexOf("TRG_014") >= 0) {
-           refind.append(j);
-        }
-        if (m_pFiffInfo->ch_names[j].indexOf("TRG_015") >= 0) {
-           refind.append(j);
-        }
-        if (m_pFiffInfo->ch_names[j].indexOf("TRG_016") >= 0) {
-           refind.append(j);
-        }
+        if (m_pFiffInfo->ch_names[j].indexOf("EEG001") >= 0) refind.append(j);
+
+        if (m_pFiffInfo->ch_names[j].indexOf("EEG002") >= 0) refind.append(j);
+
+        if (m_pFiffInfo->ch_names[j].indexOf("EEG024") >= 0) refind.append(j);
+
+        if (m_pFiffInfo->ch_names[j].indexOf("EEG025") >= 0) refind.append(j);
     }
```

Comment:
>> -        if (m_pFiffInfo->ch_names[j].indexOf("TRG_014") >= 0) {
>> -           refind.append(j);
>> -        }
>> -        if (m_pFiffInfo->ch_names[j].indexOf("TRG_015") >= 0) {
>> -           refind.append(j);
>> -        }
>> -        if (m_pFiffInfo->ch_names[j].indexOf("TRG_016") >= 0) {
>> -           refind.append(j);
>> -        }
>> +        if (m_pFiffInfo->ch_names[j].indexOf("EEG001") >= 0)
>> refind.append(j);
>> +
>> +        if (m_pFiffInfo->ch_names[j].indexOf("EEG002") >= 0)
>> refind.append(j);
>> +
>> +        if (m_pFiffInfo->ch_names[j].indexOf("EEG024") >= 0)
>> refind.append(j);
>> +
>> +        if (m_pFiffInfo->ch_names[j].indexOf("EEG025") >= 0)
>> refind.append(j);
>>      }
>
> are you using the eeg channels for the hpi tracking?
Currently yes. Later we will change to the trigger lines. Right now, the
cables are not ready.




The information in this e-mail is intended only for the person to whom it is
addressed. If you believe this e-mail was sent to you in error and the e-mail
contains patient information, please contact the Partners Compliance HelpLine at
http://www.partners.org/complianceline . If the e-mail was sent to you in error
but does not contain patient information, please contact the sender and properly
dispose of the e-mail.